components/style/properties/gecko.mako.rs

```diff
@@ -3114,33 +3114,24 @@ fn static_assert() {
                                             gecko_enum_prefix="StyleDisplay",
                                             gecko_strip_moz_prefix=False) %>
 
-    pub fn set_display(&mut self, v: longhands::display::computed_value::T) {
+    fn match_display_keyword(
+        &mut self,
```

Comment:
no need for `self` argument.

Reply:
I found we still need `self` argument because I'd like to call it from other methods inside this `impl`.
However, the `mut` is unnecessary. I'll remove it.

Reply:
You can use `Self::foo` instead.

Reply:
Ah! Yes!! I totally forgot this way! Thanks a lot!